src/semgrep_agent/semgrep.py

```diff
@@ -378,7 +376,8 @@ def _fix_head_for_github(
         if not stashed_rev:
             stashed_rev = git(["rev-parse", "HEAD"]).stdout.decode("utf-8").rstrip()
         click.echo(f"| not on head ref {head_ref}; checking that out now...", err=True)
-        git.checkout([head_ref])
+        git.checkout([head_ref], _timeout=300, _out=debug_echo, _err=debug_echo)
```

Comment:
What are the units on this?

Reply:
seconds :)
https://amoffat.github.io/sh/sections/special_arguments.html#timeout